.github/workflows/ci_v_benchmark.yml

```diff
@@ -0,0 +1,44 @@
+name: vlang benchmarks
+
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+  pull_request:
+    paths-ignore:
+      - "**.md"
+
+jobs:
+  run:
+    name: Run
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up GCC
+      uses: egor-tensin/setup-gcc@v1
+      with:
+        version: latest
+        platform: x64
```

Comment:
why is that needed?
Afaik, all the github runners already do have gcc installed.

Reply:
I will remove it, and we check it thanks!